pypdf/_cmap.py

```diff
@@ -402,19 +403,28 @@ def parse_bfchar(line: bytes, map_dict: Dict[Any, Any], int_entry: List[int]) ->
         lst = lst[2:]
 
 
-def compute_space_width(
-    ft: DictionaryObject, space_code: int, space_width: float
-) -> float:
-    sp_width: float = space_width * 2.0  # default value
-    w = []
-    w1 = {}
+def build_font_width_map(
+    ft: Union[DictionaryObject, None], map_dict: Dict[Any, Any], default_font_width: float
+) -> Dict[Any, float]:
+    font_width_map: Dict[Any, float] = {}
     st: int = 0
+    en: int = 0
+    if ft is None:
+        font_width_map["default"] = default_font_width
+        return font_width_map
+    try:
+        default_font_width = _default_fonts_space_width[cast(str, ft["/BaseFont"])] * 2.0
+    except Exception:
+        pass
     if "/DescendantFonts" in ft:  # ft["/Subtype"].startswith("/CIDFontType"):
+        # p271 PDF32000_2008 9.7.4.3 Glyph Metrics in CIDFonts
```

Comment:
I think we can omit the package number here and match it with the existing references.

Reply:
Is the following correct?

`PDF ref 1.7 9.7.4.3 Glyph Metrics in CIDFonts`